### PR TITLE
DOMA-1450 Separate accesses

### PR DESCRIPTION
**Pull request by SavelevMatthew**
Problem: payments and billingReceipts could be seen only by integration manager (administrator)
Solution: separate access of seeing billing/payment data from canManageIntegration.

(roles `canReadPayments `, `canReadBillingReceipts` was introduced during work on `Acquiring` domain, now just setting values fro existing items, changing frontend-validation)

### CODE REVIEW

File: apps/condo/pages/billing/index.tsx [hunk 1]
```diff
@@ -19,7 +17,7 @@ const BillingPage = () => {
 
     const userOrganization = useOrganization()
     const organizationId = get(userOrganization, ['organization', 'id'], '')
-    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canManageIntegrations'], false)
+    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canReadBillingReceipts'], false)
```
Comment (toplenboren) [inline apps/condo/pages/billing/index.tsx:20]: We also have a ton of access checks in billing domain. Please check /access.js

Don't we need to change them too?

**Review by toplenboren — APPROVED** [review]

**Comment by SavelevMatthew** [comment]
Forget to change access checks...moving to WIP

File: apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ async function canReadBillingIntegrationOrganizationContexts ({ authentication:
     if (user.isAdmin || user.isSupport) return true
     return {
         OR: [
-            { organization: { employees_some: { user: { id: user.id }, role: { canManageIntegrations: true }, isBlocked: false } } },
+            { organization: { employees_some: { user: { id: user.id }, role: { OR: [{ canReadBillingReceipts: true }, { canManageIntegrations: true }] }, isBlocked: false } } },
```
Comment (toplenboren) [inline apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js:16]: OR works like JOIN in graphql, please add test that:

`cant be read by organization employee without canReadBillingReceipts, but with other permissions`

Comment (toplenboren) [inline apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js:16]: OR works like JOIN in graphql, please add test that:

`cant be read by organization employee without canReadBillingReceipts, but with other permissions`

Comment (toplenboren) [reply apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js:16]: Please also add the test that:

`cant be read by deleted organization employee with with canReadBillingReceipts permission`

File: apps/condo/pages/billing/index.tsx [hunk 3]
```diff
@@ -36,7 +33,7 @@ const BillingPage = () => {
     })
 
     const PageTitle = get(currentContext, ['integration', 'billingPageTitle'], BillingTitle)
-
+    console.log(currentContext)
```
Comment (toplenboren) [inline apps/condo/pages/billing/index.tsx:36]: Please remove console.log

**Review by toplenboren — APPROVED** [review]

File: apps/condo/pages/billing/index.tsx (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/pages/billing/index.tsx]: Changed accesses in billing domain

File: apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js:16]: covered both cases with tests

**Pull request merged**